roles/unpriviledged_user/tasks/main.yml

```diff
@@ -1,62 +1,11 @@
 ---
-- name: "Creating of {{ unpriviledged_user_username }} user"
-  user:
-    name: "{{ unpriviledged_user_username }}"
-    comment: "John Vagrant"
-  become: true
-
-- name: "Add {{ unpriviledged_user_username }} to sudoers"
-  lineinfile:
-    dest: "/etc/sudoers.d/{{ unpriviledged_user_username }}"
-    state: present
-    regexp: '^{{ unpriviledged_user_username }}'
-    line: '{{ unpriviledged_user_username }} ALL=(ALL) NOPASSWD: ALL'
-    validate: 'visudo -cf %s'
-    create: yes
-  become: true
-
-- name: "Create {{ unpriviledged_user_username }} .ssh"
-  file:
-    path: "/home/{{ unpriviledged_user_username }}/.ssh"
-    owner: "{{ unpriviledged_user_username }}"
-    group: root
-    state: directory
-    mode: 0700
-  become: true
-
-- name: "Ensure public key is in authorized_keys"
-  lineinfile:
-    line:  "{{ lookup('file', unpriviledged_user_import_ssh_pub_key) }}"
-    path:  "/home/{{ unpriviledged_user_username }}/.ssh/authorized_keys"
-    create: yes
-    mode: 0600
-    owner: "{{ unpriviledged_user_username }}"
-    state: present
-  when: unpriviledged_user_import_ssh_pub_key | default(False)
-  become: true
-
-- name: "Check /home/{{ unpriviledged_user_username }}/.ssh/authorized_keys"
-  stat:
-    path: /home/{{ unpriviledged_user_username }}/.ssh/authorized_keys
-  register: authorized_keys_file
-  become: true
-
-- block:
-    - name: "Check /root/.ssh/authorized_keys"
-      stat:
-        path: /root/.ssh/authorized_keys
-      register: root_authorized_keys_file
-
-    - name: "Copy /root/.ssh/authorized_keys"
-      copy:
-        remote_src: true
-        src:  /root/.ssh/authorized_keys
-        owner: "{{ unpriviledged_user_username }}"
-        mode: 0600
-        dest: "/home/{{ unpriviledged_user_username }}/.ssh/authorized_keys"
-      when: root_authorized_keys_file.stat.exists
-
-  when:
-    - not authorized_keys_file.stat.exists
-    - unpriviledged_user_import_ssh_pub_key | default(True)
-  become: true
+- name: Print message informing the user that that the role has been renamed
+  ansible.builtin.debug:
+    msg: "Deprecation Notice: this role has moved! Please use roles/unprivileged_user instead."
+
+- name: Include moved role from new location
+  ansible.builtin.include_role:
+    name: unprivileged_user
+  vars:
+    unprivileged_user_username: "{{ unpriviledged_user_username | default('vagrant') }}"
```

Comment:
While `roles/unpriviledged_user/defaults/main.yml` is removed, the behavior is preserved by setting the default here.